### PR TITLE
zh: update doc for tiflash

### PR DESCRIPTION
**Pull request by DanielZhangQD**
### What is changed, added or deleted? (Required)

update doc for tiflash
### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/configure-cluster-using-tidbcluster.md [hunk 1]
```diff
@@ -82,3 +82,22 @@ spec:
...
+```yaml
+apiVersion: pingcap.com/v1alpha1
+kind: TidbCluster
+metadata:
+  name: basic
+spec:
+  ...
+  tiflash:
+    config:
+      config:
```
Comment (weekface) [inline zh/configure-cluster-using-tidbcluster.md:99]: two `config`?

File: zh/deploy-on-general-kubernetes.md [hunk 2]
```diff
@@ -31,6 +31,44 @@ category: how-to
...
 默认条件下，修改配置不会自动应用到 TiDB 集群中，只有在 Pod 重启时，才会重新加载新的配置文件，建议设置 `spec.configUpdateStrategy` 为 `RollingUpdate` 开启配置自动更新特性，在每次配置更新时，自动对组件执行滚动更新，将修改后的配置应用到集群中。
 
+如果要在集群中开启 TiFlash，需要在 `${cluster_name}/tidb-cluster.yaml` 文件中配置 `spec.pd.config.replication.enable-placement-rules: "true"`，并配置 `spec.tiflash`：
+
+```yaml
+  pd:
+    config:
+      ...
+      replication:
+        enable-placement-rules: "true"
```
Comment (weekface) [inline zh/deploy-on-general-kubernetes.md:41]: 这个不是会自动添加吗？

File: zh/configure-cluster-using-tidbcluster.md (hunk 1 above)
Comment (DanielZhangQD) [reply zh/configure-cluster-using-tidbcluster.md:99]: Yes, there are two config structs under the `tiflash.config`.

File: zh/deploy-on-general-kubernetes.md (hunk 2 above)
Comment (DanielZhangQD) [reply zh/deploy-on-general-kubernetes.md:41]: 对，但是全新部署的时候我觉得还是让用户配上，让用户知道这个事情。

**Pull request merged**